client/gutenberg/extensions/publicize/editor.scss

```diff
@@ -97,3 +97,13 @@ $dark-gray-500: #555d66;
 		vertical-align: middle;
 	}
 }
+
+.jetpack-publicize__connection-warning {
+	background-color: #ffd1d1;
+	border: 1px solid #eaa9a9;
```

Comment:
I assume `Notice` component will have these styles, but if not, we should use colour variables from Calypso for these.